samples/boards/nxp_s32/i3c_at24c01_eeprom/dts/bindings/atmel,at24-i3c.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright 2022 NXP
```

Comment:
this file is not needed for an i2c eeprom, the include [i2c-device.yaml] in the atmel,at24.yaml will still attach to an i3c bus as it is including the [i3c-controller.yaml]

Reply:
Yes, not really needed for the sample works. Since the eeprom is a device on i3c bus, the reg properties must contain three fields, but [i2c-device.yaml] itself doesn't mention about what does each field mean. So my idea was to create a new bindings and treat the eeprom as an i3c device

Reply:
The `-i3c` suffix signals that the binding is for a I3C device. As the EEPROM is actually a I2C device, this is going to create some confusion. Besides, the meanings of the `regs` fields are different between I2C and I3C devices. So, please just remove this.